doc/conf.py

```diff
@@ -31,7 +31,7 @@
 
 extensions.append('sphinx.ext.todo')
 todo_include_todos = True
-todo_emit_warnings = True
+todo_emit_warnings = False
```

Comment:
I'm wondering if `False` should be the default, as you have it here, and then we can flip it to true at the command-line whenever we're ready to build the document and we want to see all of the to-dos.  Thoughts?

Reply:
Sorry, this was a temporary change for me to remove the cruft while I was trying to find issues.